modules/debug.js

```diff
@@ -0,0 +1,3 @@
+var Debug = {
+	VISIONHIDE : false
```

Comment:
- Maybe make debug into a class
- VISIONHIDE should not be capitalized because it's not a constant
  - "mode" would make more sense, I feel --> "Debug.mode = true/false"